utils.js

```diff
@@ -4,6 +4,10 @@ const path = require('path');
 const got = require('got');
 const childProcess = require('child_process');
 
+const REQUESTED_VERSION = process.env.CHROMIUM_VERSION || '86.0.0';
```

Comment:
Took the opportunity to bump this to the latest dev version

Reply:
And make these vars more obviously constants that are used in a special way.  We don't have to keep the upper if y'all don't want to.